ui/components/app/confirm/info/row/expandable-row.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import classnames from 'classnames';
+import {
+  Box,
+  ButtonIcon,
+  ButtonIconSize,
+  IconName,
+} from '../../../../component-library';
+import {
+  Display,
+  IconColor,
+} from '../../../../../helpers/constants/design-system';
+import { ConfirmInfoRow, ConfirmInfoRowProps } from './row';
+
+export type ConfirmInfoExpandableRowProps = ConfirmInfoRowProps & {
+  content: React.ReactNode | string;
+  startExpanded?: boolean;
+};
+
+export const ConfirmInfoExpandableRow = (
+  props: ConfirmInfoExpandableRowProps,
+) => {
+  const { content, children, startExpanded, ...rowProps } = props;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const ref = useRef<any>();
```

Comment:
can we replace the `any` by something like `useRef<Box>()`?

Reply:
Done, although `useRef` is `T | undefined` where as `ref` is `T | null` so I had to cast.